pom.xml

```diff
@@ -122,7 +129,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>11.0.1</version>
+            <version>30.0-jre</version>
```

Comment:
upgrading guava is probably not what you want, it will cause conflicts with the jenkins core version

Reply:
which is the min version that can I use?